frontend/src/metabase/nav/containers/MainNavbar/MainNavbarView.tsx

```diff
@@ -85,6 +87,12 @@ function MainNavbarView({
   return (
     <SidebarContentRoot>
       <div>
+        <SidebarSection>
+          <HomePageLink icon="home" url="/">
+            {t`Home page`}
```

Comment:
Dunno if @kdoh disagrees but I think just "Home" is nicer. (To paraphrase Justin Timberlake channeling Sean Parker: "Drop the 'page.' Just, 'Home' … it's cleaner.")

```suggestion
            {t`Home`}
```